src/misc.jl

```diff
@@ -97,15 +124,30 @@ function findat!{T}(r::IntegerArray, a::AbstractArray{T}, b::AbstractArray{T})
     return r
 end
 
+
+"""
+    findat(a, b) -> Array
+
+Find the position within `a` for the elements in `b`.
+"""
 findat(a::AbstractArray, b::AbstractArray) = findat!(Array(Int, size(b)), a, b)
 
 
 # indicatormat
 
-# x: input elements, 
+# x: input elements,
 # c: categories
 # k: the maximum integer in x
 
+"""
+    indicatormat(x, [c,] [k,] [sparse=false]) -> Matrix
+
+Construct the indicator matrix for an array `x`. The optional argument
```

Comment:
Could you explain how this works? Are the variables to be found in rows or in columns?

Again, `c` and `k` are the same argument, only depending on the type, so you need two lines.